src/worker/users.js

```diff
@@ -218,6 +251,13 @@ class Users {
         await network.save();
         return network;
     }
+
+    async expireUserTokens() {
```

Comment:
Is this function needed anymore? Authing via tokens can now do a `AND expires_at < NOW()` in the auth queries. This removes the extra complexity of needing this loop and also keeps the tokens in the `listtokens` client command so that they can `changetoken` it if needed.